src/github.com/getlantern/lantern-mobile/app/src/main/java/org/getlantern/lantern/model/NavItem.java

```diff
@@ -1,12 +1,20 @@
 package org.getlantern.lantern.model;
 
-class NavItem {
+public class NavItem {
     String mTitle;
     int mIcon;
 
     public NavItem(String title, int icon) {
         mTitle = title;
         mIcon = icon;
     }
+
+    public String getTitle() {
+        return mTitle;
+    }
+
+    public void setTitle(String title) {
```

Comment:
We do use ``setTitle`` -- to switch the news feed menu item text from ``Try Lantern Feed`` to ``Turn off Feed``